src/jvmTest/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvReaderTest.kt

```diff
@@ -253,6 +255,44 @@ class CsvReaderTest : WordSpec() {
             }
             rows shouldBe listOf(listOf("a", "b", "c"), listOf("d", "e", "f"))
         }
+        "execute as suspending function" should {
```

Comment:
It would be better to use `suspend fun` inside `openAsync` method because these tests are for that use-case.

A `suspend fun` sample is written on Issue https://github.com/doyaaaaaken/kotlin-csv/issues/66#issue-714849682 . (`processRow` method)

Reply:
Yes I agree, but for the reason for test consistency of validating rows, I chose to use the kotlin asnyc flow.

The `collect` is a supending function, which i think cover the scenario we need.